### DISCUSSION TITLE
代码设计问题

### DISCUSSION THREAD

**Discussion by XQBxqb** [weiraneve/seckillcloud#6] [General]
对应3001端口的后台前端服务器在哪启动呢？启动项目所有商品秒杀都是已经结束了，cloud-manage调用cloud-mission模块的商品上传配置是使用qiniu，只能都这样么？

**Comment by weiraneve** [comment]
> 对应3001端口的后台前端服务器在哪启动呢？启动项目所有商品秒杀都是已经结束了，cloud-manage调用cloud-mission模块的商品上传配置是使用qiniu，只能都这样么？

这个去sql里把时间改一下就可以了；qiniu云只是上传商品照片的功能，不影响商品秒杀那些；

**Reply by XQBxqb** [reply]
还要改redis内商品的时间吧

**Reply by weiraneve** [reply]
只要重启一下对应的模块，redis里的缓存就会被更新了

**Reply by XQBxqb** [reply]
嗯，谢谢啦

**Reply by weiraneve** [reply]
不客气的～
